schema.sql

```diff
@@ -1,13 +1,26 @@
 /* Database schema to keep the structure of entire database. */
 
+
+CREATE TABLE owners (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  full_name VARCHAR(255),
+  age INTEGER
+);
+
+CREATE TABLE species (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(255)
+);
+
 CREATE TABLE animals (
-  id INTEGER PRIMARY KEY,
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
   name VARCHAR(55),
   date_of_birth DATE,
   escape_attempts INTEGER,
   neutered BOOLEAN,
-  weight_kg DECIMAL(5,2)
-);
-
-ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
+  weight_kg DECIMAL(5,2),
+  species_id INTEGER,
```

Comment:
- [ ] Instead of adding modifying the table manually by deleting or adding fields, you should use the `ALTER` command to make changes to an existing table. This prevents you from resulting into issues as you shall see in the future. It should look something like this:
```sql
   ALTER TABLE animals DROP column species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);

```

Reply:
hi @Lucas-Masaba 
I think what you asked was not on the Project requirements it should be optional suggestions.

Reply:
kindly implement the changes suggested by the previous reviewer,as it is seen in the requirement 😄 

![image](https://github.com/Khaled-Alaa-1/Vet-clinic/assets/20537712/a47245cd-5489-4bd7-a66b-018a7c12f448)